docs/index.html

```diff
@@ -15,23 +15,45 @@
     <style type="text/css">
 	    .fixed {
   		    position: fixed;
-            top: 0;
-            width: 100%;
+  		    top: 0;
   		    padding: 5px;
   		    border: 2px solid;
 	    }
 	    
 	    body, h2 {
-	        padding-top: 65px; 
+	        padding-top: 65px;
 	    }
 	    
+	    table {
```

Comment:
Don't use html tag selectors in your custom css.  Create a class that describes your change and apply that class to the element. 

Use a class selector name like " .solid-table-border"  and apply the class to the table element with class="solid-table-border" attribute.

Reply:
Should I move the css classes into the css file?